### DISCUSSION TITLE
Suggestion: Documentation rework

### DISCUSSION THREAD

**Discussion by EzraBrooks** [nasa/cFS#883] [Ideas]
I've started working on Markdown-izing documentation on my fork of this repository so it's easier to navigate via GitHub's interface. I plan to convert all plain-text, PDF, and Word documents to Markdown, but leave Doxygen docs intact since they're generated (However, on that subject, on the [CACTUS fork of cFE](https://github.com/CACTUS-Mission/cFE) I made a Travis job that automatically builds Doxygen docs and deploys them to GitHub Pages).
I can open a PR here once the conversion is complete if the cFE team would like to integrate that change in the main repo.

**Comment by skliper** [comment]
Are you still interested in contributing?  See the bundle starting at https://github.com/nasa/cFS (there's a rc-6.6.0a branch that will move to master as soon as internal testing is complete), converting to markdown is definitely a goal for all the related documentation.  We've also got a few wiki's from an internal Trac system we've also tagged for conversion.  Any help would be great!

**Comment by skliper** [comment]
CCB 20191001: General documentation (overviews, etc) markdown is preferred.  Still using doxygen for code documentation.  Preference to move away from word documents.

Preference is to actually produce the API documents using doxygen (forward work)

HTML to markdown - tool exists (maybe part of python markdown library)
pandoc does pdf to markdown
